client/app/components/Modal/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+import styles from "./styles.scss";
+import bulma from 'bulma';
+import avatar from "./IDICIA-logo-clear-center.svg";
+
+class ContactModal extends React.Component {
+    constructor () {
+      super();
+      this.state = {
+        showModal: false
+      };
+      
+      this.handleOpenModal = this.handleOpenModal.bind(this);
+      this.handleCloseModal = this.handleCloseModal.bind(this);
+    }
+    
+    handleOpenModal () {
+      this.setState({ showModal: true });
+    }
+    
+    handleCloseModal () {
+      this.setState({ showModal: false });
+    }
+    
+    render () {
+      return (
+        <div className="navbar-item animated fadeInUp">
```

Comment:
@leseanbiz fyi this looks a bit odd having the button fly into the navbar vs something like fading in. Your style choice of course.